Formula/timg.rb

```diff
@@ -0,0 +1,26 @@
+class Timg < Formula
+  desc "Terminal image and video viewer"
+  homepage "https://timg.sh/"
+  url "https://github.com/hzeller/timg/archive/refs/tags/v1.2.1.tar.gz"
+  sha256 "2ef2e1b79c3bf82e951ad35628bc0387fe8260052614a213385de82fcfa901ec"
+  license "GPL-2.0-only"
+  head "https://github.com/hzeller/timg.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "ffmpeg"
+  depends_on "graphicsmagick"
+  depends_on "jpeg-turbo"
+  depends_on "libexif"
+  depends_on "libpng"
+  depends_on "webp"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/timg", "-pq", "-g10x10", "-o/tmp/test-output.txt", test_fixtures("test.jpg")
+    assert_match "38;2;255;38;0;49m", shell_output("cut -c9-25 /tmp/test-output.txt").strip
```

Comment:
```suggestion
    system "#{bin}/timg", "-pq", "-g10x10", "-o", testpath/"test-output.txt", test_fixtures("test.jpg")
    assert_match "38;2;255;38;0;49m", (testpath/"test-output.txt").read
```
It would be good to keep the other tests too, even if we're not necessarily checking all the output.

Reply:
@carlocab Thank you for the suggestions; updated the test path, re-added the previous tests and squashed the commits, should make its way through CI now.